husky_description/urdf/husky_macro.urdf.xacro

```diff
@@ -0,0 +1,209 @@
+    <link name="${prefix}imu_link"/>
+    <joint name="${prefix}imu_joint" type="fixed">
+      <origin xyz="$(optenv HUSKY_IMU_XYZ 0.19 0 0.149)" rpy="$(optenv HUSKY_IMU_RPY 0 -1.5708 3.1416)" />
+      <parent link="${prefix}base_link" />
+      <child link="${prefix}imu_link" />
+    </joint>
+    <gazebo reference="${prefix}imu_link">
+    </gazebo>
+
+    <!-- Husky wheel macros -->
+    <xacro:husky_wheel wheel_prefix="${prefix}front_left">
+      <origin xyz="${wheelbase/2} ${track/2} ${wheel_vertical_offset}" rpy="0 0 0" />
+    </xacro:husky_wheel>
+    <xacro:husky_wheel wheel_prefix="${prefix}front_right">
+      <origin xyz="${wheelbase/2} ${-track/2} ${wheel_vertical_offset}" rpy="0 0 0" />
+    </xacro:husky_wheel>
+    <xacro:husky_wheel wheel_prefix="${prefix}rear_left">
+      <origin xyz="${-wheelbase/2} ${track/2} ${wheel_vertical_offset}" rpy="0 0 0" />
+    </xacro:husky_wheel>
+    <xacro:husky_wheel wheel_prefix="${prefix}rear_right">
+      <origin xyz="${-wheelbase/2} ${-track/2} ${wheel_vertical_offset}" rpy="0 0 0" />
+    </xacro:husky_wheel>
+
+    <xacro:husky_decorate />
+
+    <!-- <xacro:if value="$(arg laser_enabled)">
+
+      <xacro:sick_lms1xx_mount prefix="base"/>
+
+      <xacro:sick_lms1xx frame="base_laser" topic="scan" robot_namespace="$(arg robot_namespace)"/>
+
+      <joint name="laser_mount_joint" type="fixed">
+        <origin xyz="$(arg laser_xyz)" rpy="$(arg laser_rpy)" />
+        <parent link="top_plate_link" />
+        <child link="base_laser_mount" />
+      </joint>
+
+    </xacro:if> -->
+
+    <!--
+      top sensor arch; include this if we have realsense enabled
+      keep this as a property to make it easier to add multiple conditions, should we need
+      the top bar for any additional sensors in the future
+    -->
+    <!-- <xacro:property name="topbar_needed_realsense" value="$(arg realsense_enabled)" />
+    <xacro:if value="${topbar_needed_realsense}">
+      <xacro:sensor_arch prefix="" parent="top_plate_link">
+        <origin xyz="-0.35 0 0.51" rpy="0 0 -3.14"/>
+      </xacro:sensor_arch>
+    </xacro:if> -->
+
+    <!-- add the intel realsense to the topbar if needed -->
+    <!-- <xacro:if value="$(arg realsense_enabled)">
+      <link name="realsense_mountpoint"/>
+      <joint name="realsense_mountpoint_joint" type="fixed">
+        <origin xyz="0 0 0" rpy="0 0 -3.14159" />
+        <parent link="$(arg realsense_mount)"/>
+        <child link="realsense_mountpoint" />
+      </joint>
+      <xacro:intel_realsense_mount prefix="camera" topic="realsense" parent_link="realsense_mountpoint"/>
+    </xacro:if> -->
+
+    <ros2_control name="${prefix}husky_hardware" type="system">
+      <hardware>
+        <xacro:if value="$(arg is_sim)">
+          <plugin>gazebo_ros2_control/GazeboSystem</plugin>
+        </xacro:if>
+        <xacro:unless value="$(arg is_sim)">
+<!--           <plugin>husky_base/HuskyHardware</plugin> -->
```

Comment:
Should this be commented out?

Reply:
Oh you are right. This was not intended.

```suggestion
          <plugin>husky_base/HuskyHardware</plugin>
```